packages/netlify-cms-locales/src/en/index.js

```diff
@@ -176,7 +176,7 @@ const en = {
       mediaAssets: 'Media assets',
       search: 'Search...',
       uploading: 'Uploading...',
-      uploadNew: 'Upload new',
+      uploadNew: 'Upload',
```

Comment:
I think we can rename this key to `upload` instead of `uploadNew` (this will require updating all translation files).

Reply:
Done.